lib/engines/content_block_manager/app/models/content_block_manager/content_block/schema/embedded_schema.rb

```diff
@@ -40,6 +40,20 @@ def permitted_params
           end
         end
 
+        def govspeak_enabled?(field:, nested_object_key: nil)
+          return top_level_govspeak_enabled_fields.include?(field) unless nested_object_key
+
+          govspeak_enabled_fields_for_nested_object(nested_object_key).include?(field)
+        end
+
+        def govspeak_enabled_fields_for_nested_object(object_key)
+          body.dig("properties", object_key, "govspeak_enabled") || []
```

Comment:
We'll need to update this to `x-govspeak_enabled` once https://github.com/alphagov/publishing-api/pull/3493 is updated